tests/standalone_tests/test_monet_db_facade.py

```diff
@@ -0,0 +1,107 @@
+from unittest.mock import patch
+
+import pytest
+
+from mipengine import AttrDict
+from mipengine.node.monetdb_interface.monet_db_facade import db_execute_and_fetchall
+from mipengine.node.node_logger import init_logger
+from tests.standalone_tests.conftest import COMMON_IP
+from tests.standalone_tests.conftest import MONETDB_LOCALNODETMP_NAME
+from tests.standalone_tests.conftest import MONETDB_LOCALNODETMP_PORT
+from tests.standalone_tests.conftest import _restart_monetdb_container
+
+
+@pytest.fixture(scope="module", autouse=True)
+def mock_current_task():
+    current_task = AttrDict({"request": {"id": "1234"}})
+
+    with patch(
+        "mipengine.node.node_logger.node_config",
+        AttrDict(
+            {
+                "log_level": "DEBUG",
+                "role": "localnode",
+                "identifier": "localnodetmp",
+            },
+        ),
+    ), patch(
+        "mipengine.node.node_logger.task_loggers",
+        {"1234": init_logger("1234")},
+    ), patch(
+        "mipengine.node.node_logger.current_task",
+        current_task,
+    ):
+        yield
+
+    return current_task
+
+
+@pytest.fixture(autouse=True, scope="session")
+def patch_node_config():
+    with patch(
+        "mipengine.node.monetdb_interface.monet_db_facade.node_config",
+        AttrDict(
+            {
+                "monetdb": {
+                    "ip": COMMON_IP,
+                    "port": MONETDB_LOCALNODETMP_PORT,
+                    "database": "db",
+                    "username": "monetdb",
+                    "password": "monetdb",
+                },
+                "celery": {
+                    "tasks_timeout": 60,
+                    "run_udf_task_timeout": 120,
+                    "worker_concurrency": 1,
+                },
+            }
+        ),
+    ):
+        yield
+
+
+@pytest.mark.slow
+def test_execute_and_fetchall_success(
+    mock_current_task,
+    patch_node_config,
```

Comment:
nit: `mock_current_task` and `patch_node_config` don't need to be declared there since `autouse=True`.